### ISSUE TITLE
Empty profile after upgrade

### ISSUE THREAD

**Issue by raphgilles** [YunoHost-Apps/pixelfed_ynh#43]
Hello,

I have an issue after the upgrade to the 0.9

All looks working fine : the homepage, the public timeline, or the discover page. But the profile page keep empty.
Here is an exemple with my own profile : https://pixelfed.fr/raph

Any idea ?

Thank you :)

EDIT : I think I have the same issue with the single publications page : https://pixelfed.fr/p/raph/36803376792604672

**Comment by yalh76** [comment]
Hi @raphgilles, 

All seems ok, except ynh tile that don't load but that's normal....

**Comment by yalh76** [comment]
you have that problem with you profil or all profiles ?

**Comment by raphgilles** [comment]
I have this problem on all profiles. And on all publication pages.

**Comment by yalh76** [comment]
> 
> 
> I have this problem on all profiles. And on all publication pages.

I have tried to install previous version but as Yunohost as been updated, previous install don't work

**Comment by raphgilles** [comment]
So, must I downgrade to .8.6 and wait ?

**Comment by yalh76** [comment]
> 
> 
> I have this problem on all profiles. And on all publication pages.

have you got something in the nginx logs ?

> 
> 
> So, must I downgrade to .8.6 and wait ?

The actual version of pixelfed_ynh is 0.9.0

Have you got something in the nginx logs ?

**Comment by raphgilles** [comment]
Yes I know, but I can restore my backup.

Here is the nginx error log : https://paste.yunohost.org/mimekaloje

**Comment by yalh76** [comment]
log content is logical.

maybe you can try first to upgrade to last pixelfed_ynh using: `yunohost app upgrade -u https://github.com/YunoHost-Apps/pixelfed_ynh`

and if it don't work restore the backup or ask help at pixelfeld :///

**Comment by raphgilles** [comment]
When I try to do that to the 0.8.6 it says me "no apps to upgrade".

**Comment by kay0u** [comment]
The command is:

`yunohost app upgrade pixelfed -u https://github.com/YunoHost-Apps/pixelfed_ynh`

**Comment by raphgilles** [comment]
Ok. So the problem is not solved. I think I will wait the future versions. If the issue persist, I will contact pixelfed.

**Comment by yalh76** [comment]
Ok then, let us know, maybe it's an issue in pixelfed_ynh package

**Comment by raphgilles** [comment]
Just to update the topic.

A friend broke his account after sending a large file.
https://pixelfed.fr/julienlecomte

So now, I can work on the issue without breaking all my instance ah ah !

And if you have any idea, I'm curious. Because actually I don't know where to search.

I tried php artisan cache:clear / optimize / purge ect... with no result.

**Comment by raphgilles** [comment]
So.

Let me explain you how finally I made my upgrade !

1. I installed a fresh clean install on an other server.
2. I upgraded my main instance.
3. I copied all the files of the clean install to my main instance. But not the .env and the storage folder.

And voilà ! It's works. The profile that I linked before is always down. Finaly it's an other problem. But all my instance works now. Thank you for the help !
